src/environment.rs

```diff
@@ -9,6 +9,15 @@ pub struct Environment {
     handle: raw::SQLHENV,
 }
 
+/// Struct holding information available on a driver.
+///
+/// Can be obtained via `Environment::drivers`
+#[derive(Clone, Debug)]
+pub struct DriverInfo {
+    pub description: String,
+    pub attributes: Vec<(String, String)>,
```

Comment:
Those attributes could be a hashmap - may be it will be more comfortable for end user?